CMake/FindNGHTTP2.cmake

```diff
@@ -0,0 +1,26 @@
+include(SelectLibraryConfigurations)
+include(FindPackageHandleStandardArgs)
+
+find_path(NGHTTP2_INCLUDE_DIR "nghttp2/nghttp2.h"
+          HINTS ${NGHTTP2_DIR}/include ${NGHTTP2_DIR}/inc)
+
+find_library(NGHTTP2_LIBRARY_RELEASE NAMES NGHTTP2LIB

```

Comment:
Name should be lowercase to work on case-sensitive filesystems (Linux).

@bradking is this appropriate use of `select_library_configurations` in a FindXXX.cmake file?